python/ray/tests/test_object_spilling_2.py

```diff
@@ -0,0 +1,506 @@
+import json
+import os
+import random
+import platform
+import subprocess
+import sys
+from collections import defaultdict
+
+import numpy as np
+import pytest
+import ray
+from ray.test_utils import wait_for_condition, run_string_as_driver
+from ray.tests.test_object_spilling import is_dir_empty, assert_no_thrashing
+
+
+@pytest.mark.skipif(
+    platform.system() == "Windows", reason="Failing on Windows.")
+def test_delete_objects(object_spilling_config, shutdown_only):
```

Comment:
Is there any possible better names for the two test files?

Reply:
I thought about multi node & single node, but the number of tests are not balanced. Couldn't really think of good naming here (since it was very hard to find two categories that can evenly split tests). Do you have any suggestion?